src/repositories/user.repository.ts

```diff
@@ -7,14 +8,25 @@ export const getAdminUser = () => {
 };
 
 export const findUserByUserName = async (username: string) => {
```

Comment:
basically the same question to `findUserByUserName` method. I don't see any reasons to create it, and would suggest to keep this logic inside `findUserData` method

Reply:
I know that's @VitaliiDenysyuk issue, but could you fix it in your PR?